sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-resource-server-by-filter/src/main/java/com/azure/spring/sample/aad/controller/TodoListController.java

```diff
@@ -45,6 +47,18 @@ public TodoListController() {
         return model;
     }
 
+
+    @RequestMapping({"/"})
+    public ModelAndView index() {
+        ModelAndView model = new ModelAndView("index");
+        model.addObject("aad_clientId", aadAuthenticationProperties.getClientId());
+        model.addObject("aad_tenantId", aadAuthenticationProperties.getTenantId());
+        model.addObject("aad_redirectUri", Optional
```

Comment:
code style?

Reply:
I think @saragluna  means change `aad_clientId` to `aad_client_id`.